test/interop/C/exportString/badFormalIntent.chpl

```diff
@@ -1,6 +1,6 @@
+export proc badFormalIntentDefault(s: string) {}
 
-
-
+export proc badFormalIntentConstRef(const ref s: string) {}
 
 export proc badFormalIntentIn(in s: string) {}
```

Comment:
Should the versions that work currently be commented or removed?

Reply:
I'll take that to mean you'd like me to remove them... so I'll do that.

Reply:
I'm good either way.  I think the file has been altered in the past due to switching which ones are supported (or at least a similar test has been), so leaving them around might save future time?

Reply:
But hopefully we only extend the supported intents in the future